docs/_posts/examples/.eslintrc

```diff
@@ -5,7 +5,8 @@
     "MapboxGeocoder": true,
     "MapboxDirections": true,
     "turf": true,
-    "d3": true
+    "d3": true,
+    "Uint8Array": true
```

Comment:
Yeah, you're right. And `"parserOptions": { "ecmaVersion": 5 }` doesn't work either. 😞